dd-trace-api/src/main/java/datadog/trace/api/Config.java

```diff
@@ -126,6 +126,9 @@
 
   public static final int DEFAULT_JMX_FETCH_STATSD_PORT = 8125;
 
+  public static final int DEFAULT_DB_STATEMENT_MAX_LENGTH = 1024;
+  public static final String DB_STATEMENT_MAX_LENGTH = "db.statement.max.length";
```

Comment:
Do you mind moving this up with the other system properties and documenting it in the readme?